lib/flux/app-state.js

```diff
@@ -439,15 +439,27 @@ export const actionMap = new ActionMap({
       },
     },
 
+    syncNote: {
+      creator({ noteBucket, noteId }) {
+        console.log('sync the note');
```

Comment:
if we do this we should add `debug()` - taking out the log statement altogether is also fine